### PR TITLE
Address some CR, and use NLTK for word tokenize in preprocess

### PR DESCRIPTION
**Pull request by KeDengMS**
This is a patch for existing BiDAF model

### CODE REVIEW

File: bidaf/README.md [hunk 1]
```diff
@@ -61,17 +57,15 @@ The model has 2 outputs.
...
 start = np.asscalar(answer[0])
 end = np.asscalar(answer[1])
-print(cw[start:end+1])
+print([w.encode() for w in cw[start:end+1].reshape(-1)])
 ```
 
 For this testcase, it would output
 ```
-[['brown']].
+[b'brown'].
```
Comment (prasanthpul) [inline bidaf/README.md]: where does the "b" come from?

File: bidaf/README.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ This model is a neural network for answering a query about a given context parag
 
  |Model        |Download  |Checksum|Download (with sample test data)|ONNX version|Opset version|Accuracy |
 |-------------|:--------------|:--------------|:--------------|:--------------|:--------------|:--------------|
-|BiDAF  |[41.5 MB](https://onnxzoo.blob.core.windows.net/models/opset_9/bidaf/bidaf.onnx) | [MD5](https://onnxzoo.blob.core.windows.net/models/opset_9/bidaf/bidaf-md5.txt) |[37.3 MB](https://onnxzoo.blob.core.windows.net/models/opset_9/bidaf/bidaf.tar.gz)|1.5 |9 |EM of 68.1 in SQuAD v1.1 |
+|BiDAF  |[41.5 MB](https://onnxzoo.blob.core.windows.net/models/opset_9/bidaf/bidaf.onnx) | [MD5](https://onnxzoo.blob.core.windows.net/models/opset_9/bidaf/bidaf-md5.txt) |[37.3 MB](https://onnxzoo.blob.core.windows.net/models/opset_9/bidaf/bidaf.tar.gz)|1.5 |ONNX 9, ONNX.ML 1 |EM of 68.1 in SQuAD v1.1 |
```
Comment (prasanthpul) [inline bidaf/README.md]: With this combo of opsets, according to https://github.com/onnx/onnx/blob/master/docs/Versioning.md I think we can say this model is ONNX 1.4

File: bidaf/README.md (hunk 1 above)
Comment (KeDengMS) [reply bidaf/README.md]: The string may contain unicode characters that cannot be print. The change to print above uses str.encode() to output the bytes array, thus b'...'

---
In reply to: [283202312](https://github.com/onnx/models/pull/159#discussion_r283202312) [](ancestors = 283202312)

File: bidaf/README.md (hunk 2 above)
Comment (KeDengMS) [reply bidaf/README.md]: Thanks, changed to 1.4

---
In reply to: [283203132](https://github.com/onnx/models/pull/159#discussion_r283203132) [](ancestors = 283203132)

**Pull request merged**